test/lib.sh

```diff
@@ -1284,10 +1292,10 @@ subjectAltName = DNS:clicon.org
 EOF
 
     # Generate server key
-    openssl genpkey -algorithm RSA -out $srvkey  || err "Generate server key"
+    openssl genpkey -quiet -algorithm RSA -out $srvkey  || err "Generate server key"
```

Comment:
'-quiet' or '-batch' not supported on older versions
olof@nuc1:~/src/clixon/test$ openssl -quiet
Invalid command '-quiet'; type "help" for a list.
olof@nuc1:~/src/clixon/test$ openssl version
OpenSSL 1.1.1  11 Sep 2018
olof@nuc1:~/src/clixon/test$ uname -a
Linux nuc1 4.15.0-211-generic #222-Ubuntu SMP Tue Apr 18 18:55:06 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
olof@nuc1:~/src/clixon/test$ lsb_release --all
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic
olof@nuc1:~/src/clixon/test$

Reply:
Done